ptz-explainer.md

```diff
@@ -31,22 +31,25 @@ steering the camera to face a speaker for instance.
 Without it, users have to install native apps today to control camera PTZ.
 Presently, zoom is supported as a media track constraint behind the camera
 permission. The plan is to group pan, tilt and zoom together behind a PTZ
-separate permission.
+separate permission which can be requested in a single `getUserMedia()` call
+with the camera permission.
 
 ## Control camera pan/tilt
 
 We would like to add `pan` and `tilt` to the existing property set of media
 track capabilities, constraints and settings. These new constraints apply to the
 live video feed. Those will also be used in `getUserMedia()` to express whether a
 website wants to control camera PTZ functionality. In other words, it will be
-used to request the PTZ permission, a separate camera permission, in a single
-call. If the selected/connected camera does not support PTZ though, the PTZ
-permission won’t be requested and may fallback to the camera permission. If the
-user denies the PTZ permission, it may also fallback to the camera permission.
+used to request the PTZ permission, a separate permission, in a single
+`getUserMedia()` call, along the camera permission. If the selected/connected
+camera does not support PTZ though or user blocks solely the PTZ permission, the
+UA may fall back to the camera permission.
```

Comment:
I don't think that is a good recommendation as people might plug and unplug cameras with such support. I do that as I share cameras between computers

Reply:
That's why I've set "may fall back" as it's up to the UA. If it turns out to be unsuccessful, we should update the explainer indeed.